packages/lab/src/carousel/Carousel.tsx

```diff
@@ -1,170 +1,59 @@
 import {
-  Button,
-  GridLayout,
-  RadioButton,
-  RadioButtonGroup,
-  makePrefixer,
-  useIcon,
+  type ResponsiveProp,
+  resolveResponsiveValue,
+  useBreakpoint,
   useId,
 } from "@salt-ds/core";
-import { useComponentCssInjection } from "@salt-ds/styles";
-import { useWindow } from "@salt-ds/window";
-import { clsx } from "clsx";
-import {
-  type ChangeEventHandler,
-  Children,
-  type HTMLAttributes,
-  type ReactElement,
-  forwardRef,
-  useEffect,
-} from "react";
-import { DeckLayout } from "../deck-layout";
-import { useSlideSelection } from "../utils";
-import type { CarouselSlideProps } from "./CarouselSlide";
-
-import carouselCss from "./Carousel.css";
-
-const withBaseName = makePrefixer("saltCarousel");
+import { type HTMLAttributes, forwardRef } from "react";
+import { CarouselProvider } from "./CarouselContext";
 
 export interface CarouselProps extends HTMLAttributes<HTMLDivElement> {
   /**
    * The initial Index enables you to select the active slide in the carousel.
    * Optional, default 0.
    **/
-  initialIndex?: number;
-  /**
-   * The animation when the slides are shown.
-   * Optional. Defaults to `slide`
-   **/
-  animation?: "slide" | "fade";
-  /**
-   * If this props is passed it will set the aria-label with value to the carousel container.
-   * Optional. Defaults to undefined
-   */
-  carouselDescription?: string;
+  activeSlideIndex?: number;
```

Comment:
If I set activeSlideIndex, to 2 then when the page loads, you get scroll movement, as it scrolls to the activeSlideIndex.